src/test/unit/service/publicationService.test.ts

```diff
@@ -76,7 +76,7 @@ describe('Publication service', () => {
 
   it('should return list types', () => {
     const listTypes = publicationService.getListTypes();
-    expect(listTypes.size).to.equal(13);
+    expect(listTypes.size).to.equal(14);
```

Comment:
Strikes me that we should remove one of these list type counts as doing it twice is a pain.